base/applications/rapps/gui.cpp

```diff
@@ -278,16 +278,20 @@ BOOL CMainWindow::RemoveSelectedAppFromRegistry()
     if (MessageBoxW(szMsgText, szMsgTitle, MB_YESNO | MB_ICONQUESTION) == IDYES)
     {
         CInstalledApplicationInfo *InstalledApp = (CInstalledApplicationInfo *)m_ApplicationView->GetFocusedItemData();
-        LSTATUS Result = InstalledApp->RemoveFromRegistry();
-        if (Result != ERROR_SUCCESS)
+        if (InstalledApp)
```

Comment:
Just handle the `if (!InstalledApp)` case here first; this will allow to not have this indentation increase (and reduce the patch).

Reply:
Good point !